### PR TITLE
withdrawn

### PR DESCRIPTION
**Pull request by gcolvin**

### CODE REVIEW

**Comment by MicahZoltu** [comment]
While I don't agree with this decision, an author *can* choose to withdraw at any time.  I would encourage you to reconsider while we wait for CI to finish @gcolvin!

File: EIPS/eip-2315.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ---
 eip: 2315
 title: Simple Subroutines for the EVM
-status: Draft
+status: Withdrawn
```
Comment (lightclient) [inline EIPS/eip-2315.md:4]:
```suggestion
status: Withdrawn
```

**Comment by gcolvin** [comment]
Thanks Micah.  There is nothing left for me to reconsider :(

**Comment by eip-automerger** [comment]
Hi! I'm a bot, and I wanted to automerge your PR, but couldn't because of the following issue(s):

 - Trying to change EIP 2315 state from Draft to Withdrawn

**Pull request merged**